cli/src/builders/make/customRules.ts

```diff
@@ -0,0 +1,102 @@
+import { warningOut } from "../../cli";
+import { ObjectType, Targets } from "../../targets";
+import { getFiles } from "../../utils";
+import { readFileSync } from "fs";
+import * as path from "path";
+import { MakeProject } from ".";
+
+/**
+ * Scan for all rules.mk files and read attributes and custom
+ * dependencies into the targets.
+ */
+export function readAllRules(targets: Targets, project: MakeProject) {
+  // getFiles is case insensitive
+  const rulesFiles = getFiles(targets.getCwd(), `**/rules.mk`);
```

Comment:
FYI, if you are in a case sensitive file system, the case is `Rules.mk`

Reply:
the `getFiles` API has a property passed into it to ignore case. That applies on every filesystem is runs on.

Reply:
OK, that explains why the tests work :-)